src/libraries/Common/src/System/HexConverter.cs

```diff
@@ -314,6 +316,13 @@ public static bool TryDecodeFromUtf16_Vector128(ReadOnlySpan<char> chars, Span<b
                     even = AdvSimd.ShiftLeftLogical(even, 4).AsInt16();
                     output = AdvSimd.AddSaturate(even, odd).AsByte();
                 }
+                else if (PackedSimd.IsSupported)
+                {
+                    Vector128<byte> shiftedNibbles = PackedSimd.ShiftLeft(nibbles, 4);
+                    //Vector128<byte> zipped = PackedSimd.BitwiseSelect(nibbles, shiftedNibbles, Vector128.Create((ushort)0xFF00).AsByte());
```

Comment:
The commented line looks identical?

Reply:
I reverted back once I fixed BitwiseSelect but failed to remove the comment